libs/sr-formula/src/data/char/index.ts

```diff
@@ -38,9 +39,15 @@ function handleCharacterGen(
 }
 
 const data: TagMapNodeEntries[] = [
-  nonTrailblazerCharacterKeys
-    .filter((k) => k !== 'March7th')
-    .flatMap(handleCharacterGen),
-  March7th,
+  nonTrailblazerCharacterKeys.flatMap((key) => {
+    switch (key) {
+      case 'March7th':
```

Comment:
nit: probably not sustainable once we add more sheets, you can decide when you want to address this

Reply:
Once I add the generator in the next PR, this will be nuked anyways